python/vyos/config.py

```diff
@@ -253,6 +253,9 @@ def show_config(self, path=[], default=None, effective=False):
             path = " ".join(path)
         try:
             out = self._run(self._make_command('showConfig', path))
+            # ugly but can not see a better way to handle this
+            if out == 'Configuration under specified path is empty\n':
```

Comment:
This is fragile. I see your point, but let me check if I can make cli-shell-api exit with 1 in such cases.

Reply:
T2027 is now handling this in a cleaner way